### ISSUE TITLE
Reduce number of otterdog-app comments

### ISSUE THREAD

**Issue by mbarbero** [eclipse-csi/otterdog#286]
Currently, when a PR is created, the otterdog-app generates at least five different comments (e.g., see [this PR](https://github.com/eclipse-californium/.eclipsefdn/pull/1)).

For some, if not all, of these messages, it would be better to update the initial comment rather than creating a new one.

The challenge lies in finding the correct comment to edit. The strategy I've used in the past is to begin the body of the comment with a header enclosed in comment tags (similar to HTML ones: `<!-- -->`). [These won't be visible in the rendered comment](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments), but the content returned by the API will include these strings. By standardizing the first line of the body with similar content, it becomes easier to identify the specific message to update.

**Comment by netomi** [comment]
This is already done, try to edit a comment from the bot and you will see the raw data.

In general, if the same type of comment is generated again, the now deprecated one of the same type is minimized.
The initial amount of comments is too much I agree, the reason the are separate is because separate tasks are being executed and can be requested via commands.

**Comment by mbarbero** [comment]
Sorry for the oversight. When you "separate tasks," I assume they are executed in parallel, which could introduce race conditions when editing the same comment, correct?

**Comment by netomi** [comment]
I fixed a bug where the check-sync comment was always added after a PR got updated. In order to reduce the noise this check is only executed if the last check is more than 1h old. The idea is that this check verifies that the config is in sync with the live settings.

**Comment by netomi** [comment]
these tasks are executed concurrently, we could change the logic to have less comments that are generated from the app and edit these comments rather than minimizing and creating new ones.
